jetty/content.md

```diff
@@ -38,6 +38,24 @@ Webapps can be [deployed](https://www.eclipse.org/jetty/documentation/current/qu
 
 For older EOL'd images based on Jetty 7 or Jetty 8, please follow the [legacy instructions](https://wiki.eclipse.org/Jetty/Howto/Deploy_Web_Applications) on the Eclipse Wiki and deploy under `/usr/local/jetty/webapps` instead of `/var/lib/jetty/webapps`.
 
+## Configuration
+
+The configuration of the jetty server can be reported by running with the --list-config option:
+```console
+$ docker run -d %%REPO%%:9 --list-config
+```
+Configuration such as parameters and additional modules may also be passed in via the command line. For example
+```console
+$ docker run -d %%REPO%%:9 --modules=jmx jetty.threadPool.maxThreads=500
+```
+To update the server configuration in a new docker image, the Dockerfile may enamble additional modules with RUN commands like:
```

Comment:
`s/docker/Docker/` `s/enamble/enable/`

Use code formatting for "Dockerfile" and "RUN"